support-frontend/assets/pages/[countryGroupId]/student/helpers/discountDetails.ts

```diff
@@ -57,30 +58,44 @@ export function getDiscountSummary({
 	)}`;
 }
 
-type StudentDiscount = {
-	fullPriceWithCurrency: string;
+export type StudentDiscount = {
 	amount: number;
 	periodNoun: string;
+	discountPriceWithCurrency: string;
+	fullPriceWithCurrency: string;
 	promoCode?: string;
 	promoDuration?: string;
 	discountSummary?: string;
-	discountPriceWithCurrency?: string;
 };
 
+function isStudent(
+	geoId: GeoId,
+	ratePlanKey: ActiveRatePlanKey,
+	productKey: ActiveProductKey,
+	promotion?: Promotion,
+): boolean {
+	const isOneYearStudent = ratePlanKey === 'OneYearStudent' && geoId !== 'au';
```

Comment:
I wonder if we need the geo check here. Could you get here with a rate plan of `OneYearStudent` and a geoId of `au`? Also if we do want to exclude by region, shouldn't there be other regions in here? (Global student offer is only us, uk and ca I think).

Reply:
Think AU student is a `Monthly` ratePlanKey not a `OneYearStudent` which is quite different and not recurring (as you spotted in the Ts&Cs). I deliberately kept this wide by using `geoId !== 'au'` so we could add regions in the future without having to update this portion of the code - is that wise/ok?